dags/veda_data_pipeline/groups/discover_group.py

```diff
@@ -21,9 +21,9 @@ def discover_from_s3_task(ti, event={}, **kwargs):
         **event,
         **ti.dag_run.conf,
     }
-    if not kwargs.get('dag_run').external_trigger:
-        config["last_successful_execution"] = kwargs.get("prev_start_date_success")
-
+    last_successful_execution = kwargs.get("prev_start_date_success")
```

Comment:
don't we want to be able to overwrite this for manual trigger? 👀

Reply:
I was thinking if we need to trigger it manually we can rerun the first run (the one without any previous run). We can start with this and see how often we need a manual run and if we need to come with "more" elegant solution.

Reply:
i can see a scenario where we want to run all files since a specific time to avoid having to ingest ALL THE FILES again

Reply:
Even with the "if" statement you will be ingesting ALL the FILES. We probably need to think about a regular expression to limit the file discovery when we run it manually

Reply:
i was thinking we could provide our own `last_successful_execution` as part of the config when running manually

Reply:
The new push includes the key "process_from_yyyy_mm_dd" to the event. Right now we are supporting only year-month-day. 
If we want to process all the files modified since March 05 2024 we need to add to the event
`
{
"process_from_yyyy_mm_dd": "2024-03-05"
...
}
`
This will force the discovery to discover the files updated after the specified date